src/elements/content-uploader/ContentUploader.js

```diff
@@ -191,6 +193,11 @@ class ContentUploader extends Component<Props, State> {
     componentDidMount() {
         this.rootElement = ((document.getElementById(this.id): any): HTMLElement);
         this.appElement = this.rootElement;
+        const { files, allowPrepopulateFiles } = this.props;
+        // allowPrepopulateFiles is a prop used to pre-popluate files without clicking upload button.
+        if (allowPrepopulateFiles && files && files.length > 0) {
+            this.addFilesToUploadQueue(files, this.upload);
+        }
```

Comment:
nit: minor simplification - `files` is defaulted to an empty array
```suggestion
        if (allowPrepopulateFiles && files.length) {
            this.addFilesToUploadQueue(files, this.upload);
        }
```

Reply:
it was failing for me in the ci check, thats why i added it. maybe while compiling it needs this check but once the element loads this check wont fail as it is defaulted to [].